sklearn_hierarchical_classification/tests/fixtures.py

```diff
@@ -89,3 +94,29 @@ def make_classifier_and_data(
     )
 
     return clf, (X, y)
+
+
+def make_clothing_graph(root=ROOT):
+    """Create a mock hierarchy of clothing items."""
+    G = DiGraph()
+    G.add_edge(root, "Mens")
+    G.add_edge("Mens", "Shirts")
+    G.add_edge("Mens", "Bottoms")
+    G.add_edge("Mens", "Jackets")
+    G.add_edge("Mens", "Swim")
+
+    return G
```

Comment:
added these fixture factory methods to use the clothing hierarchy that was used to reproduce the bug in #31